tests/integration/targets/user_info/tasks/main.yaml

```diff
@@ -0,0 +1,52 @@
+- name: user_info
+  block:
+    - debug:
+       msg: Skipping for now...(https://jira.linode.com/browse/TPT-2939)
```

Comment:
Is there more graceful way to skip the test?

Reply:
Nope as of now , same has been followed in other test as well. i.e account_info